grape-swagger.gemspec

```diff
@@ -14,7 +14,7 @@ Gem::Specification.new do |s|
   s.license     = 'MIT'
 
   s.required_ruby_version = '>= 2.4'
-  s.add_runtime_dependency 'grape', '~> 1.3.0'
+  s.add_runtime_dependency 'grape', '>= 1.3.0', '< 1.5.0'
```

Comment:
use '~> 1.3' instead, it will do the magic

Reply:
gosh..yep :palm: 
since your comment is longer and more content than the actual change, well, "sorry" for "contributing "  :smile: thanks